addon/components/hyper-table/index.hbs

```diff
@@ -161,7 +161,7 @@
   {{/if}}
 
   {{#if (and (eq this._collection.length 0) (not this.loadingData))}}
-    <div class="hypertable__state hypertable__state--empty">
+    <div class="hypertable__state hypertable__state--empty" {{did-insert (action "centerElement")}}>
```

Comment:
centerElement should really be renamed to something more precise
`scrollToEmptyState` `focusEmptyState` etc